crates/backend/src/ast.rs

```diff
@@ -26,6 +27,26 @@ pub struct Program {
     pub typescript_custom_sections: Vec<String>,
     /// Inline JS snippets
     pub inline_js: Vec<String>,
+    /// Path to wasm_bindgen
+    pub wasm_bindgen: Path,
+    /// Path to wasm_bindgen_futures
+    pub wasm_bindgen_futures: Path,
+}
+
+impl Default for Program {
+    fn default() -> Self {
+        Self {
+            exports: Default::default(),
+            imports: Default::default(),
+            linked_modules: Default::default(),
+            enums: Default::default(),
+            structs: Default::default(),
+            typescript_custom_sections: Default::default(),
+            inline_js: Default::default(),
+            wasm_bindgen: syn::parse_quote! { wasm_bindgen },
+            wasm_bindgen_futures: syn::parse_quote! { wasm_bindgen_futures },
```

Comment:
This should have been `::wasm_bindgen` and `::wasm_bindgen_futures` for added proc-macro hygiene. Unfortunately that would be a breaking change.

Reply:
Mentioning it here for reference: we can make that change alongside https://github.com/rustwasm/wasm-bindgen/pull/3372.